src/tty/windows.rs

```diff
@@ -296,6 +296,13 @@ impl ConsoleRenderer {
             pos,
             &mut _count,
         ));
+        check!(wincon::FillConsoleOutputAttribute(
+            self.handle,
+            attr,
+            length,
+            pos,
+            &mut _count
```

Comment:
```suggestion
            &mut _count,
```

Reply:
`cargo fmt` doesn't add a comma !

Reply:
That is because this is inside a macro, so it doesn't know it is allowed to do that.

Reply:
Indeed !